tests/postgres_tests/models.py

```diff
@@ -192,3 +192,7 @@ class HotelReservation(PostgreSQLModel):
     end = models.DateTimeField()
     cancelled = models.BooleanField(default=False)
     requirements = models.JSONField(blank=True, null=True)
+
+
+class JSONModel(models.Model):
```

Comment:
`JSONModel` will crash on databases that don't support `JSONField`:
```suggestion
class JSONModel(PostgreSQLModel):
```
Can we reuse `HotelReservation`?

Reply:
Must've missed that field on `HotelReservation`, thanks!

Reply:
Thanks, also `JSONField` is now supported on all backends. Can we move tests for key transforms to `tests/indexes`?

Reply:
Moved them `schema/indexes`. There was no model available with `JSONField` so I had to reintroduce `JSONModel`.. Also discovered that MySQL do not support functional indexes where the return value is JSON: https://dev.mysql.com/doc/mysql-errors/8.0/en/server-error-reference.html#error_er_functional_index_on_json_or_geometry_function

So we ignore the `KeyTransform` on MySQL.